README.md

```diff
@@ -1 +1,131 @@
-# setup-linter
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My-portfolio-project: <a name="about-project"></a>
+
+> The so called Linter setup project is created using pure Html5 and Css3. In this project I have configured linter for html and css.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- BUILT WITH -->
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Major languages: 
+    HTML, CSS3
+- Technologies used: 
+    Git, GitHub, Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+ 
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Simple
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project.
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git and Code editor.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+``git clone `https://https://github.com/eyobyoseph/Hello-world.git`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Eyob Getachew 
+
+- GitHub: [eyobyoseph](https://github.com/eyobyoseph)
+- Twitter: [@eyobyoseph1](https://eyobyoseph1s://twitter.com/)
+- LinkedIn: [Eyob Getachew](linkedin.com/in/eyob-yoseph-53ba27244)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://https://github.com/eyobyoseph/Hello-world/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project please give a ⭐️ to this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank [Microverseinc](https://github.com/microverseinc).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This link is still broken. That is because the licence file you added is called **MIT.md** but you are refering to a file called **LICENSE**. Please refer to the correct file 😄